src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -95,11 +95,11 @@ export default function DateTimeSelectPage() {
       getAppointmentSlots(
         moment(preferredDate)
           .startOf('month')
-          .format('YYYY-MM-DD'),
+          .format('YYYY-MM-DDTHH:mm:ssZ'),
```

Comment:
What was the reason for making these format changes? I think this will be hard to test, so I'd like to stick with the current format if we can.

Reply:
Moved the formatting to the `getSlots` function